WebRoot/js/zimbraMail/abook/view/ZmEditContactView.js

```diff
@@ -1099,6 +1099,7 @@ ZmEditContactViewImage = function(params) {
 	this.addListener(DwtEvent.ONMOUSEOVER, new AjxListener(Dwt.addClass, [el,DwtControl.HOVER]));
 	this.addListener(DwtEvent.ONMOUSEOUT, new AjxListener(Dwt.delClass, [el,DwtControl.HOVER]));
 	this.addListener(DwtEvent.ONMOUSEUP, new AjxListener(this, this._chooseImage));
+	el.onkeydown = ZmEditContactViewImage._onKeyDown.bind(this);
```

Comment:
can we use addListener function instead? onkeydown looks very old syntax

Reply:
Fixed.